### ISSUE TITLE
Purple boosted icon is very hard to see, even in the "High contrast" theme.

### ISSUE THREAD

**Issue by dilinger** [mastodon/mastodon#19596]
### Steps to reproduce the problem

1. Switch to Mastodon's "high contrast" theme in the web interface.
2. Boost a toot
3. Watch as the color changes very slightly from a dark grey to a dark purple, and then wonder if it was actually boosted or not.

### Expected behaviour

An obvious color change that shows that I boosted a toot

### Actual behaviour

A very slight color change, almost indistinguishable unless you look very closely

### Detailed description

As I mentioned on https://mastodon.online/@Andres4NY/109264235018179907  , the color scheme for a boost is really difficult to see. On a normal theme, fine... But in the "high contrast" mode, I'd especially expect some colors that actually contrasted. I'm not pretending to be some kind of artist or designer, so I don't have a suggested color scheme, but something other than dark purple would be a lot more usable.  The orange color that the favorite icon changes to a lot more obvious, for example.

### Specifications

Firefox 102.4.0esr, on Debian 11

Mastodon.online's web ui (which is likely the latest version?)

**Comment by FediVideos** [comment]
Yes, agreed. Almost impossible to see whether a post is boosted. The old colour scheme was much clearer.

**Comment by dilinger** [comment]
Unfortunately, this is not actually fixed. It still looks the same, even after the server update.

**Comment by joenepraat** [comment]
> Unfortunately, this is not actually fixed. It still looks the same, even after the server update.

I can confirm this. Also in the normal dark theme it's very hard to distinguish from other icons. Only under trending posts, where you can see the amount of boosts, it's clear that you boosted this (because of the numbers).

![afbeelding](https://user-images.githubusercontent.com/41594439/202226451-5dd94095-fdd7-4872-8d29-154039eb4729.png)

Unfortunately I can't fix this using server side CSS, because it's a inline SVG. So please fix this ASAP @Gargron (in both themes), because it effects a lot of people.

**Comment by joenepraat** [comment]
BTW: What I don't understand is why the boost icon uses a SVG. The other icons do not and as you can see the boost icon doesn't scale well. I now managed to change this icon to a green color, what is more neutral and easier for the eyes IMO.

**Comment by joenepraat** [comment]
@Gargron @ClearlyClaire This is still not fixed in 4.1.0. I get reports from several users (with bad vision or older persons) that can't see when a post is boosted. Also I can't hardly see it. Because this is an accessibility issue, I don't understand why this isn't already fixed?

![afbeelding](https://user-images.githubusercontent.com/41594439/222446750-fc5d6bfd-b332-46b2-bffe-167da484f5c3.png)

**Comment by dilinger** [comment]
I have a vague memory (but no good way to search for it) of @Gargron saying in a mastodon thread about this bug that someone needs to come up with a better color scheme. To that end, I bookmarked (and then promptly forgot about until now) the following toot:

https://robot.rodeo/@alex/109318117430319312

That google doc has a nice list of ("AAA") options, although it's pretty clear that using the same icon colors for dark and light mode/backgrounds very much does not work. For a black background, I personally like #D5D5D1 for the non-clicked icons, and a darker orange like #FFBF00 for the clicked icons (boost and like).

**Comment by FediVideos** [comment]
This is still an issue, it's not at all obvious a post has been boosted.

The Favourite turns yellow when you click it. Why can't the boost turn a similarly bright colour such as green?

**Comment by sam1919** [comment]
It's even worse now on mastodon.online because the icon has changed.
Please check if this post has been boosted or not. It's really hard to see.

![post](https://github.com/mastodon/mastodon/assets/581826/3a6bbe41-d23a-49a2-9035-1cdd53bb9700)

**Comment by ClearlyClaire** [comment]
We have some ideas on how to solve that, but it'll require a bit more work. See #27924 and #27665

**Comment by diondiondion** [comment]
This has been fixed in the meantime; the boost icon now changes shape as well as colour when active.

**Comment by todoneunl** [comment]
Unfortunately the contrast is still not good enough for some people with eye sight issues. Setting the theme in high contrast mode doesn't change this.

**Comment by diondiondion** [comment]
> Unfortunately the contrast is still not good enough for some people with eye sight issues. Setting the theme in high contrast mode doesn't change this.

Can you post a more specific example?

As far as I can see, the active icon passes WCAG's AAA contrast requirements in both light (7.41) and dark (8.27) mode, even when not in high-contrast mode. If color blindness is an issue, the changed shape (square in centre) should provide a hint that the button was pressed.

**Comment by todoneunl** [comment]
@diondiondion Let's take this example (see image). I had at least two persons that have problems to see enough contrast, especially when using desktop night light setting (reduces the blue's after sunset). I think the WCAG doesn't look to that. One user has successfully changed it to another color (green), so maybe it's an idea to change the color in the high contrast mode as well. Oh and the square in the centre is indeed a good addition, but probably not enough for some people. Don't get me wrong, I think the accessibility improvements are great, but not every person is the same. So there will always be people for which it don't work as expected. 

Edit: I tried to paste a screenshot with the nightlight setting on (Ubuntu), but that gives the same result is below. 

<img width="727" height="383" alt="Image" src="https://github.com/user-attachments/assets/c308b927-3291-41d4-ac8d-3a81ab55b5b1" />

**Comment by diondiondion** [comment]
I see, so it's more about the contrast between inactive and active buttons. I'll reopen the ticket and rename the topic to be more general as I think it should be addressed more holistically.

**Comment by todoneunl** [comment]
> I see, so it's more about the contrast between inactive and active buttons. I'll reopen the ticket and rename the topic to be more general as I think it should be addressed more holistically.

Thank you. Yes, it's about the difference between inactive and active buttons. And because the contrast of the boost icon is lower, than e.g. the favorite icon, some (older) people on my servers at least have problems seeing if they have boosted a post. This is especially the case when using the night light mode, because that reduces the blue colours.
